src/Domain/FunctionsOfTime/PiecewisePolynomial.hpp

```diff
@@ -76,12 +77,17 @@ class PiecewisePolynomial : public FunctionOfTime {
   // NOLINTNEXTLINE(google-runtime-references)
   void pup(PUP::er& p) override;
 
- private:
   template <size_t LocalMaxDeriv>
   friend bool operator==(  // NOLINT(readability-redundant-declaration)
```

Comment:
This should still be private I think? (I don't actually understand why it _can_ be private, but that's what all other classes do)